src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -368,6 +382,14 @@ public static BigIntUnsignedHandlingMode parse(String value, String defaultValue
             .withValidation(CommonConnectorConfig::validateTopicName)
             .withDescription("Overrides the topic.prefix used for the data change topic.");
 
+    public static final Field OVERRIDE_SCHEMA_CHANGE_TOPIC = Field.create("override.schema.change.topic")
```

Comment:
@jpechane Following up on discussion [here](https://github.com/debezium/debezium-connector-vitess/pull/210/files#r1825801913), we do still need this config. The [SchemaTopicNamingStrategy](https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/schema/SchemaTopicNamingStrategy.java#L40-L42) does not override `schemaChangeTopic` so it defaults to the implementation in [AbstractTopicNamingStrategy](https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/schema/AbstractTopicNamingStrategy.java#L124) ie just using topic.prefix. We do need a custom one. I updated the docstring in the `vitess.TableTopicNamingStrategy` to provide more context on this (ie with vitess-connectors we can have mulitple connectors for the same database, so need to be able to override this).

Reply:
You provide `io.debezium.connector.vitess.TableTopicNamingStrategy`. So why can't you provide two strategies that would differ in shcema handling? Or if you want to make it configurable, there is a `configure` method at teh strategy class level so you can move the config field into the strategy itself in the same way as is done at `io.debezium.schema.AbstractTopicNamingStrategy`. That way it will be configurable yet the field will not be defined at connector level.

Reply:
If we separate the strategies (one does the override data change topic, one does the schema change topic override) then afaik we can't combine two topic naming strategies in the config so we cannot use both of them. If vitess-connectors want to enable this feature and they have multiple connectors per database they would need to use both (see this [test](https://github.com/debezium/debezium-connector-vitess/pull/215/files#diff-1b9011e9a90b9250c8885c36b838ac867f597b3726cf037ec1fa0aefb47d8221R264-R265) for demonstration).

I will go with that second option of moving the configs into the vitess.TableTopicNamingStrategy class.